doc/code-docs/locales/en/LC_MESSAGES/checkpoint.po

```diff
@@ -36,11 +37,93 @@ msgstr ""
 
 #: ../../source/checkpoint.rst:6
 msgid "InternLM支持启动时自动加载最新的模型备份，并在接收信号退出训练时自动进行模型备份。"
-msgstr "InternLM supports automatic loading of latest ckpt at startup and automatic model checkpointing at signal quit. "
+msgstr "InternLM supports automatic loading of latest ckpt at startup and automatic model checkpointing at signal quit."
 
 #: ../../source/checkpoint.rst:9
-msgid "Checkpointing"
+msgid "CheckpointManager"
+msgstr ""
+
+#: ../../source/checkpoint.rst:11
+msgid ""
+"``CheckpointManager`` "
+"是InternLM负责进行模型加载和保存的工具类，其会使用config文件中的ckpt字段的初始化参数字典初始化自身的参数，目前相关的参数有："
+msgstr ""
+"CheckpointManager is the utility class within InternLM responsible for "
+"model loading and saving. It initializes its own parameters using the "
+"initialization parameter dictionary from the 'ckpt' field in the config "
+"file. Currently, the relevant parameters are as follows"
+
+#: ../../source/checkpoint.rst:13
+#, fuzzy
+msgid "``enable_save_ckpt``: 是否开启检查点存储功能（不影响检查点加载）。参数类型 ``bool``，必选参数。"
+msgstr ""
+"``enable_save_ckpt``: Whether to enable checkpoint storage functionality "
+"(does not affect checkpoint loading). Parameter type: `bool`, it is a "
+"required parameter."
+
+#: ../../source/checkpoint.rst:15
+#, fuzzy
+msgid "``save_ckpt_folder``: 检查点存储路径，参数类型 ``str``，默认为： ``None``，在开启检查点存储功能时为必选参数。"
+msgstr ""
+"``save_ckpt_folder``: Checkpoint storage path. Parameter type: ``str``. "
+"This is a required parameter when enabling checkpoint storage "
+"functionality."
+
+#: ../../source/checkpoint.rst:17
+#, fuzzy
```

Comment:
This line need to be deleted to make english document rendered normally.

References: https://github.com/sphinx-doc/sphinx/issues/9618